synonyms/en.yml

```diff
@@ -1207,23 +1207,27 @@ housed, housing:
   :status: Approved
   :analyzed: housed, housing
 hunt, hunted, hunter, hunting:
-  :notes: 
+  :notes:
   :status: Candidate
   :analyzed: hunt, hunted, hunter, hunting
 ia, iowa:
-  :notes: 
+  :notes:
   :status: Candidate
   :analyzed: iowa, ia
 id, identification, identity, identify:
   :notes: Approved (abbreviations, noun-verb pairs). Prefering ID in this sense, and not the state abbreviation. DPM 3/29/2018.
   :status: Approved
   :analyzed: id, identif, ident, identifi
+ig, oig:
+  :notes: Approved (acronyms). Inspector General = Office of the Inspector General. DPM 7/6/18
+  :status: Approved
+  :analyzed:
```

Comment:
@dawnpm , I'm not too familiar w/ the Punchcard YAML. Is there a reason you did not set `id, oig` as `analyzed`? All the other acronyms in this file set `analyzed` to be the acronyms, i.e.:
```
ffel, ffelp:
  :notes: Approved (acronyms) for Federal Family Education Loan (FFEL) Program. AFF
    10/8/14
  :status: Approved
  :analyzed: ffel, ffelp
```
Can you confirm and/or test?

Reply:
@MothOnMars At some point I remember James telling me that it was optional, plus I don't have access to a tool that would help me see the analyzed form.